src/component/mxgraph/ShapeConfigurator.ts

```diff
@@ -13,17 +13,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { MxGraphFactoryService } from '../../service/MxGraphFactoryService';
+
 import { mxgraph } from 'ts-mxgraph';
 import { ShapeBpmnElementKind } from '../../model/bpmn/shape/ShapeBpmnElementKind';
 import { EndEventShape, StartEventShape, ThrowIntermediateEventShape, CatchIntermediateEventShape } from './shape/event-shapes';
 import { ExclusiveGatewayShape, ParallelGatewayShape, InclusiveGatewayShape } from './shape/gateway-shapes';
 import { ReceiveTaskShape, ServiceTaskShape, TaskShape, UserTaskShape } from './shape/task-shapes';
 
+declare const mxClient: typeof mxgraph.mxClient;
+declare const mxShape: typeof mxgraph.mxShape;
+declare const mxCellRenderer: typeof mxgraph.mxCellRenderer;
+declare const mxSvgCanvas2D: any;
```

Comment:
there are inconsistencies concerning type from factory - I leave it as any as we will pass too hungtcs library and it will be resolved